x-pack/plugins/security/public/views/nav_control/nav_control.html

```diff
@@ -10,7 +10,7 @@
   <global-nav-link
     kbn-route="'/logout'"
     icon="'plugins/security/images/logout.svg'"
-    tooltip-content="formatTooltip('Logout')"
-    label="'Logout'"
+    tooltip-content="formatTooltip(logoutLabel)"
```

Comment:
@joshdover I added localization for tooltip as well, but I didn't find `formatTooltip` function. I think it should be similar to `accountTooltip` https://github.com/elastic/kibana/pull/26011/files#diff-02abf9793c68e526c53c333ee2301584R41
Is this issue known?

Reply:
Yeah I'm not seeing that function anywhere either. I agree it seems like it should just be accountTooltip

Reply:
Don't you mind if I just replace `formatTooltip` with `accountTooltip` in this PR?